### PR TITLE
Feature: Make timestamp optional #658

### PR DESCRIPTION
**Pull request by adeelhussain**
Making timestamp optional, simply pass `--timestamp false` will skip the timestamp from the migration file and set the `timestamps: false` in model
Referenced to #658

### CODE REVIEW

File: src/commands/model_generate.js [hunk 1]
```diff
@@ -17,6 +17,12 @@ exports.builder =
           type: 'string',
           demandOption: true
         })
+        .option('timestamp', {
```
Comment (sushantdhiman) [inline src/commands/model_generate.js]: `timestamps`

File: src/assets/migrations/create-table.js [hunk 2]
```diff
@@ -11,21 +11,30 @@ module.exports = {
...
         },
 
-        <% attributes.forEach(function(attribute) { %>
+        <% attributes.forEach(function(attribute, index) { %>
           <%= attribute.fieldName %>: {
             type: Sequelize.<%= attribute.dataFunction ? `${attribute.dataFunction.toUpperCase()}(Sequelize.${attribute.dataType.toUpperCase()})` : attribute.dataValues ? `${attribute.dataType.toUpperCase()}(${attribute.dataValues})` : attribute.dataType.toUpperCase() %>
-          },
+          }
+          
+          <% if(attributes.length -1 !== index) { %>
```
Comment (sushantdhiman) [inline src/assets/migrations/create-table.js]: space after `if`

File: src/assets/migrations/create-table.js [hunk 3]
```diff
@@ -11,21 +11,30 @@ module.exports = {
...
+            ,
+          <% } %>
+
         <% }) %>
 
-        <%= createdAt %>: {
-          allowNull: false,
-          type: Sequelize.DATE
-        },
+        <% if(typeof(createdAt) != "undefined"){ %>
```
Comment (sushantdhiman) [inline src/assets/migrations/create-table.js]: `if (typeof createdAt !== undefined)`

File: src/commands/model_generate.js (hunk 1 above)
Comment (adeelhussain) [reply src/commands/model_generate.js]: @sushantdhiman done!

**Comment by adeelhussain** [comment]
Well i think its not a problem but i have removed that trailing comma.

**Comment by sushantdhiman** [comment]
Build/Tests are still failing

**Comment by ephys** [comment]
In case OP becomes active with this PR again:

Options should be false by default. If we want timestamps to generate by default, the cli option to disable them should be named `skip-timestamps` or `no-timestamps` and default to false.

I'm also closing the related documentation PR as this one is closed. (https://github.com/sequelize/sequelize/pull/11410)
If activity resumes, we can reopen both PRs

**Pull request closed without merging**